tests/firstLoginTests.spec.js

```diff
@@ -0,0 +1,40 @@
+const { test, expect } = require("@playwright/test");
+
+test.describe("two simple tests", () => {
+  test.beforeEach(async ({ page }) => {
+    const emailInput = page.getByPlaceholder("Username");
+    const passwordInput = page.getByPlaceholder("Password");
+    const loginButton = page.locator("#login-button");
+
+    await page.goto("https://www.saucedemo.com/");
```

Comment:
you can add this url to playwright config file

Reply:
Additionally as we have more time this week. Lets try to play and run tests on other browsers